src/page_range.rs

```diff
@@ -52,7 +52,7 @@ impl FromGlibContainerAsVec<gtk_sys::GtkPageRange, *mut gtk_sys::GtkPageRange> f
 
         let mut res = Vec::with_capacity(num);
         for i in 0..num {
-            res.push(PageRange(ptr::read(ptr.offset(i as isize))));
+            res.push(PageRange(ptr::read(ptr.add(i))));
```

Comment:
I didn't know this method. :o

Reply:
The violated clippy lint even points you at it :D